zookeeper-client/zookeeper-client-c/src/zookeeper.c

```diff
@@ -5111,11 +5111,11 @@ int zoo_add_auth(zhandle_t *zh,const char* scheme,const char* cert,
 
 static const char* format_endpoint_info(const struct sockaddr_storage* ep)
 {
-    static char buf[134] = { 0 };
+    static __thread char buf[134] = { 0 };
```

Comment:
Do we need a macro here to accommodate single thread version and different compilers ?

cc @ztzg

Reply:
`__thread` will do nothing in st mode, generating the same asm as without it